spec/support/expire_cookies.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+module ExpireCookies
+  def expire_cookies
```

Comment:
I don't like the idea of using whole big gem instead of little module, if it been something complicated, required tests - gem it is, but in this case gem just increase complicity but gives nothing. May be I wrong.